gcp-bundle/LICENSE

```diff
@@ -220,129 +220,123 @@ License: Apache 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+Group: com.google.auth  Name: google-auth-library-credentials  Version: 1.28.0
 License: BSD New license - http://opensource.org/licenses/BSD-3-Clause
 
 --------------------------------------------------------------------------------
 
-Group: com.google.auth  Name: google-auth-library-oauth2-http  Version: 1.18.0
+Group: com.google.auth  Name: google-auth-library-oauth2-http  Version: 1.28.0
 License: BSD New license - http://opensource.org/licenses/BSD-3-Clause
 
 --------------------------------------------------------------------------------
 
-Group: com.google.auto.value  Name: auto-value-annotations  Version: 1.10.1
+Group: com.google.auto.value  Name: auto-value-annotations  Version: 1.11.0
 Project URL: https://github.com/google/auto/tree/master/value
 License: Apache 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.cloud  Name: google-cloud-core  Version: 2.20.0
+Group: com.google.cloud  Name: google-cloud-core  Version: 2.45.0
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.cloud  Name: google-cloud-core-grpc  Version: 2.20.0
+Group: com.google.cloud  Name: google-cloud-core-grpc  Version: 2.45.0
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.cloud  Name: google-cloud-core-http  Version: 2.20.0
+Group: com.google.cloud  Name: google-cloud-core-http  Version: 2.45.0
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.cloud  Name: google-cloud-storage  Version: 2.22.5
+Group: com.google.cloud  Name: google-cloud-storage  Version: 2.43.2
 Project URL: https://github.com/googleapis/java-storage
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.code.findbugs  Name: jsr305  Version: 3.0.2
-Project URL: http://findbugs.sourceforge.net/
-License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
-
---------------------------------------------------------------------------------
-
-Group: com.google.code.gson  Name: gson  Version: 2.10.1
+Group: com.google.code.gson  Name: gson  Version: 2.11.0
 Project URL: https://github.com/google/gson/gson
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.18.0
+Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.33.0
```

Comment:
I wonder if this should be 2.35.1? https://github.com/apache/iceberg/blob/1.7.x/gradle/libs.versions.toml#L41